site/source/docs/getting_started/downloads.rst

```diff
@@ -198,8 +202,21 @@ The package manager can do many other maintenance tasks ranging from fetching sp
 .. _downloads-uninstall-the-sdk:
 
 Uninstalling the Emscripten SDK
-========================================================
+===============================
+
+If you want to remove the whole SDK, just delete the directory containing the
+SDK.
+
+It is also possible to :ref:`remove specific tools in the SDK using emsdk
+<emsdk-remove-tool-sdk>`.
+
+Using the Docker image
+======================
+
+The entire Emscripten SDK is also available in the form a `docker image
+<https://hub.docker.com/r/emscripten/emsdk>`_.  For example::
 
-If you want to remove the whole SDK, just delete the directory containing the SDK.
+  docker run --rm -v $(pwd):/src -u $(id -u):$(id -g) \
+    emscripten/emsdk emcc helloworld.cpp -o helloworld.js
```

Comment:
Just in case - does this command actually work? We mount into `/src`, I don't remember if it's the default dir, or we need to `cd` first.

Reply:
/src is the default working directory within the container yes.  No need to `cd`: https://github.com/emscripten-core/emsdk/blob/147abfe46c7828cb413e27cea082bfe47335c2f2/docker/Dockerfile#L132

Reply:
And yes, this command as written does work at least with `1.40.0` and `1.39.20` and `latest` (a.k.a `1.40.0`).    

We had some issues with file permissions that were fixes yesterday so it wasn't working as non-root for a while, but it is now.

Reply:
Great, thanks for confirming.